plugins/palo_alto_cortex_xdr/help.md

```diff
@@ -52,6 +52,42 @@ Example input:
 
 ### Actions
 
+#### Get File Quarantine Status
+
+This action is used to get File Quarantine Status.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|endpoint_id|string|None|True|Endpoint ID|None|SOMEENDPOINT|
+|file_hash|string|None|True|File Hash|None|275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f|
+|file_path|string|None|True|File path|None|powershell.exe|
+
+Example input:
+
+```
+{
+  "endpoint_id": "SOMEENDPOINT",
```

Comment:
Both here and in the endpoint_id the example needs to be somewhat realistic. So like 
"example-host" or something to that effect.